main.cpp

```diff
@@ -76,45 +103,73 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
 } 
 
-/*
- 1)
- */
+ float carCost(float downPayment, float carPrice, int numCars)
+ {
+     ignoreUnused(downPayment, carPrice, numCars = 1);
+     return {};
+ }
 
-/*
- 2)
- */
 
-/*
- 3)
- */
+  char password(char password, char username)
+ {
+     ignoreUnused(password, username);
+     return {};
+ }

```

Comment:
- Fix the indentation.  
`char` is not lined up with the left edge of the editor
- `password` is not an action, it is a noun
choose actions when creating functions.  Functions DO things, so name them based on what they do.
- [ ] revise